### PR TITLE
fix!: runtype var name must be static

### PR DESCRIPTION
**Pull request by nis65**
## Description

This (together with the other MR) was needed on my setup to get first boot to work at all.

## Dependencies

* This should should be merged at the same time as this [MR in the basics role](https://github.com/projectpotos/ansible-role-potos_basics/pull/27)
* This replaces the dirty and closed MR #45 (for the same issue)

This is a breaking change as users that made use of `potos_basics_ansible_runtype_var_name` will lose that
feature (well, I doubt that it was usable - i.e. if you could set it to anything else than `potos_runtype` at all before :smile:)

### CODE REVIEW

**Comment by fadnincx** [comment]
Will merge that after corresponding PR is merged in basics (breaking change).

**Comment by nis65** [comment]
Analysis on running system, all affected roles
~~~
potos-iso-builder: change needed
potos-apt: no change needed
     1  ./ansible/roles/apt/defaults/main.yml:### Example to deinstall apt packages, at potos_runtype setup only
     2  ./ansible/roles/apt/tasks/apt_package.yml:        potos_runtype == setup
potos_basics: change needed
     3  ./ansible/host_vars/localhost/potos_basics.yml:potos_basics_ansible_runtype_var_name: "potos_runtype"
ansible-plays-potos: no change needed
     4  ./ansible/playbook.yml:    potos_runtype: 'daily'
     5  ./ansible/molecule/default/converge.yml:        - 'ansible-playbook -e "potos_runtype=daily" /potos/prepare.yml'
     6  ./ansible/molecule/default/converge.yml:        - 'ansible-playbook -e "potos_runtype=daily" /potos/playbook.yml'
ansible-specs; change needed
     7  ./ansible/specs/vars/potos_basics.yml:potos_basics_ansible_runtype_var_name: "potos_runtype" 
     8  ./ansible/specs/files/templates/requirements.yml.j2:{% if potos_runtype == 'setup' %}
     9  ./ansible/templates/requirements.yml.j2:{% if potos_runtype == 'setup' %}

~~~

**Comment by nis65** [comment]
These roles already use a hardcoded variable name, no need to change:

* potos-apt
* ansible-plays-potos

**Comment by nis65** [comment]
Hi @fadnincx - I was going through all my roles to check where I can switch back to the upstream **projectpotos** role and found that this MR is still missing. And then I saw the reason: the commit msg was still missing the `!` for the breaking change. Amended and force pushed.

**Review by fadnincx — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**
